common/src/main/java/com/microsoft/identity/common/internal/cache/SharedPreferencesFileManager.java

```diff
@@ -67,6 +68,13 @@
     private static final ConcurrentMap<String, SharedPreferencesFileManager> objectCache =
             new ConcurrentHashMap<String, SharedPreferencesFileManager>(16, 0.75f , 1);
 
+    /**
+     * Clear all cache data from sharedPreferences.
+     */
+    public static void clearAll() {
+        objectCache.clear();
```

Comment:
This is a bit confusing - when is one supposed to call this?

Reply:
Documented.